static/panes/compiler.ts

```diff
@@ -316,6 +316,10 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
 
         this.revealJumpStack = [];
 
+        if (this.compiler) {
+            this.compiler.activeOverrides = state.overrides;
```

Comment:
Best to store this somewhere else like the global https://github.com/compiler-explorer/compiler-explorer/blob/main/static/global.ts#L30 , or in local storage if you want it to persist beyond refreshes